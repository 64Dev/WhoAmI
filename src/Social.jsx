function Social() {
    return (
        <div className="social-icons">
            <a href="https://github.com/6ix4" target="_blank" className="icon-button github" rel="noreferrer">
                <i className="fa-brands fa-github-alt fa-bounce fa-xl"></i>
            </a>
            <a href="mailto:exact@six4.dev" className="icon-button email">
                <i className="fa-duotone fa-envelope-open-text fa-xl"></i>
            </a>
        </div>
    );
}

export default Social;