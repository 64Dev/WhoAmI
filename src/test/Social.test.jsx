import {it, expect} from 'vitest';
import Social from "../Social.jsx";

it("Social", () => {
    expect(<Social />).toMatchSnapshot('<Social />');
} );