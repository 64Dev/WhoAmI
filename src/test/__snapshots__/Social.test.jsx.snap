// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Social > <Social /> 1`] = `<Social />`;
